### ISSUE TITLE
v-bind use dymaic import get errors

### ISSUE THREAD

**Issue by Volankey** [vuejs/eslint-plugin-vue#1246]
**Tell us about your environment**
* **ESLint version:** 6.7.2
* **eslint-plugin-vue version:** 6.1.2
* **Node version:** v13.3.0

**Please show your full configuration:**

```json
module.exports = {
  root: true,
  env: {
    node: true,
  },
  extends: ['@vue/standard', 'plugin:vue/recommended'],
  rules: {
    'no-console': process.env.NODE_ENV === 'production' ? 'error' : 'off',
    'no-debugger': process.env.NODE_ENV === 'production' ? 'error' : 'off',
    'comma-dangle': [
      'error',
      {
        arrays: 'always',
        objects: 'always',
        imports: 'never',
        exports: 'never',
        functions: 'never',
      },
    ],
    'comma-spacing': [
      'error',
      {
        before: false,
        after: true,
      },
    ],
    'space-before-function-paren': 0,
    'no-shadow': ['error'],
    'standard/object-curly-even-spacing': [0, 'either'],
    'standard/array-bracket-even-spacing': [0, 'either'],
    'standard/computed-property-even-spacing': [2, 'even'],
    'standard/no-callback-literal': [2, ['cb', 'callback']],
    'array-bracket-spacing': 'off',
    'vue/max-attributes-per-line': [
      2,
      {
        singleline: 20,
        multiline: {
          max: 1,
          allowFirstLine: false,
        },
      },
    ],
    'vue/no-parsing-error': [
      2,
      {
        'x-invalid-end-tag': false,
      },
    ],
  },
  parserOptions: {
    parser: 'babel-eslint',
    "ecmaVersion": 8,
    "sourceType": "module"
  },
}

```

**What did you do?**	

```vue
 <IconRender
            style="vertical-align:middle;"
            :icon="()=>import('ui/lib/icons/ios-more')"
            :size="30"
            color="#333"
          />
```

**What did you expect to happen?**
no error

**What actually happened?**

{
	"resource": "xxx.vue",
	"owner": "_generated_diagnostic_collection_name_#1",
	"severity": 8,
	"message": "\n[vue/valid-v-bind]\n'v-bind' directives require an attribute value.",
	"source": "eslint-plugin-vue",
	"startLineNumber": 60,
	"startColumn": 13,
	"endLineNumber": 60,
	"endColumn": 56
}
{
	"resource": "xxx.vue",
	"owner": "_generated_diagnostic_collection_name_#1",
	"severity": 8,
	"message": "\n[vue/no-parsing-error]\nParsing error: Unexpected token import.",
	"source": "eslint-plugin-vue",
	"startLineNumber": 60,
	"startColumn": 24,
	"endLineNumber": 60,
	"endColumn": 24
}

**Comment by ota-meshi** [comment]
Thank you for this issue.

I use `babel-eslint` but it doesn't reproduce your problem. Could you share your minimal repository to reproduce the issue?

**Comment by Volankey** [comment]
> t it doesn't reproduce your problem. Could you share your minimal repository to reproduce the issue?

I found that I use `npm run lint` will not report an error, but it will report an error in vscode

![image](https://user-images.githubusercontent.com/23287363/87644982-6eafa400-c77f-11ea-9354-ecd2485bbb35.png)

![image](https://user-images.githubusercontent.com/23287363/87645034-81c27400-c77f-11ea-92f4-9ff71be03542.png)

![image](https://user-images.githubusercontent.com/23287363/87645172-b0d8e580-c77f-11ea-993e-edaab9250fd0.png)

![image](https://user-images.githubusercontent.com/23287363/87645204-ba624d80-c77f-11ea-9652-150149814d9e.png)

**Comment by ota-meshi** [comment]
Now I understand.
You need to add the following settings to `.vscode/settings.json`.

```json
{
   "vetur.validation.template": false
}
```

https://vuejs.github.io/vetur/linting-error.html#error-checking
https://eslint.vuejs.org/user-guide/#visual-studio-code

**Comment by Volankey** [comment]
> Now I understand.
> You need to add the following settings to `.vscode/settings.json`.
> 
> ```json
> {
>    "vetur.validation.template": false
> }
> ```
> 
> https://vuejs.github.io/vetur/linting-error.html#linting
> https://eslint.vuejs.org/user-guide/#visual-studio-code

Thank you very much for your prompt reply! (どうもありがとう)
